administrator/language/de-DE/com_joomlaupdate.ini

```diff
@@ -76,13 +76,18 @@ COM_JOOMLAUPDATE_UPDATE_LOG_FILE="Datei %s erfolgreich heruntergeladen."
 COM_JOOMLAUPDATE_UPDATE_LOG_FINALISE="Installation abschließen."
 COM_JOOMLAUPDATE_UPDATE_LOG_INSTALL="Neue Version wird jetzt installiert."
 COM_JOOMLAUPDATE_UPDATE_LOG_START="Update wurde vom Nutzer %2$s (%1$s) gestartet. Die frühere Version war %3$s."
+COM_JOOMLAUPDATE_UPDATE_LOG_UPLOAD="Update-Datei hochladen"
 COM_JOOMLAUPDATE_UPDATE_LOG_URL="Updatedatei von %s wird heruntergeladen."
+COM_JOOMLAUPDATE_UPDATE_LOGGING_TEST_FAIL="Das Logging funktioniert nicht. Sicherstellen, dass der Log-Ordner beschreibbar ist, und erneut versuchen. Der Loggingtest ist mit der Meldung fehlgeschlagen: %s"
```

Comment:
```suggestion
COM_JOOMLAUPDATE_UPDATE_LOGGING_TEST_FAIL="Die Protokollierung funktioniert nicht. Sicherstellen, dass das Protokollverzeichnis beschreibbar ist, und erneut versuchen. Der Protokollierungstest ist mit der Meldung fehlgeschlagen: %s"
```
![image](https://github.com/joomlagerman/joomla/assets/66922325/45754d5e-d6ef-467f-b9a3-44179c92a6b2)

In der Konfiguration wird `Protokollierung` verwendet
![image](https://github.com/joomlagerman/joomla/assets/66922325/7104cad6-1f13-444c-80d7-653604fdcec6)